src/wallet/hdchain.h

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2020 The PIVX Core developers
+// Distributed under the MIT software license, see the accompanying
+#ifndef PIVX_HDCHAIN_H
+#define PIVX_HDCHAIN_H
+
+#include "key.h"
+
+namespace HDChain {
+    namespace ChangeType {
```

Comment:
nit: why the nested namespaces? Wouldn't a `HDChangeType` enum suffice?

Reply:
did it in this way to have a common `HDChain` space for any future global param and utility functions.